docs/src/content/docs/getting-started/learning.md

```diff
@@ -97,49 +145,79 @@ const ctx = createCtx()
 
 const aAtom = atom(1, 'aAtom')
 const bAtom = atom(2, 'bAtom')
+const cAtom = atom((ctx) => ctx.spy(aAtom) + ctx.spy(bAtom), 'cAtom')
 
 const setParams = action((ctx, a: number, b: number) => {
   console.log(`change a=${a}, b=${b}`)
   aAtom(ctx, a)
   bAtom(ctx, b)
 }, 'setParams')
 
-const cAtom = atom((ctx) => ctx.spy(aAtom) + ctx.spy(bAtom), 'cAtom')
-
 ctx.subscribe(cAtom, (c) => {
   const a = ctx.get(aAtom)
   const b = ctx.get(bAtom)
   console.log(`${a} + ${b} = ${c}`)
 })
 
 setParams(ctx, 10, 12)
-// logs: change a=10, b=12
-// logs: 10 + 12 = 22
+// change a=10, b=12
+// 10 + 12 = 22
 
 setParams(ctx, 10, 12)
-// logs: change a=10, b=12
-// does not log the result since it did not change
-
-bAtom(ctx, 4)
-// logs: 10 + 4 = 14
+// change a=10, b=12
+// (does not log the result because it has not changed)
 ```
 
 As we see here the subscribe callback only called with both a and b values changed.
-This is called transactions and they enable us to control how frequent consumer of state is notified of some changes.
+This is called "transactions". They help to reduce the number of subscriber calls, and avoid the creation of unwanted intermediate states
 
-<!--
+### Async actions
 
-## Schedule to use asyncronous actions
-Sometimes you need to call something to be able to change something. Reatom does not support asyncronous transactions yet.
-Instead there's a special mechanism of queues that enble us to do something.
+Creating asynchronous actions is also possible, but keep in mind that async operations must be called inside `ctx.schedule` callback
 
-TODO example with ctx.schedule and ctx.schedule -1
+```ts
+import { createCtx, atom, action } from '@reatom/framework'
 
-You can take a deeper dive in queues here
+export const dataAtom = atom(null)
 
--->
+export const fetchData = action(async (ctx) => {
+  const data = await ctx.schedule(() => {
+    const response = fetch('https://jsonplaceholder.typicode.com/todos/1')
+    const payload = response.json()
+  })
+  dataAtom(ctx, data)
+})
+```
+
+### Actions nesting
+You can call actions from other actions. And asynchronous actions will return the promise
+```ts
+import { action, atom } from '@reatom/core'
+
+export const todoAtom = atom(null)
+export const isLoadingAtom = atom(false);
+
+export const fetchTodo = action(async (ctx) => {
+  const response = await ctx.schedule(() => fetch('https://jsonplaceholder.typicode.com/todos/1'))
+  return await response.json();
+})
+
+export const loadTodo = action(async (ctx) => {
+  try {
+    isLoadingAtom(ctx, true)
+    const data = await ctx.schedule(() => fetchTodo(ctx))
```

Comment:
Nope, you need to take context from callback here, so you can just call ctx.schedule(fetchTodo) to do that implicitly